workers/loc.api/sample.js

```diff
@@ -0,0 +1,15 @@
+const { Api } = require('bfx-wrk-api')
+
+class Sample extends Api {
+  async sampleEndpoint (space, token, cb) {
```

Comment:
Does the method name determine the action in the query when making a request?

Reply:
yes